<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleEditDialog</class>
 <widget class="QDialog" name="RuleEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Label_1">
       <property name="text">
        <string>ルール名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputRuleName"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>指定方法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inputMatchMode"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageFilters">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>キーワード</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>条件タイプ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddFilter">
           <property name="text">
            <string>フィルターを追加</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRegex">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_2">
           <property name="text">
            <string>正規表現</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="inputRegex"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Label_3">
           <property name="text">
            <string>置換モード</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="inputReplaceMode"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Label_4">
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="text">
        <string>置換先文字列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputReplaceText"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ルールプレビュー</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputPreviewText">
       <property name="placeholderText">
        <string>対象文字列を入力...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPreview">
       <property name="text">
        <string>変換-&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputPreviewText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>置換結果</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>適用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>キャンセル</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
